### PR TITLE
Text updates in submission process

### PR DESCRIPTION
**Pull request by viviancan**
Refs #4086 Text updates in submission process

### CODE REVIEW

File: app/views/team_submissions/pieces/development_platform.en.html.erb [hunk 1]
```diff
@@ -99,15 +99,15 @@
           id: :team_submission_thunkable_account_email %>
 
         <%= f.label :thunkable_project_url,
-          "What is the URL to your Thunkable project?",
+          "What is the URL to your Thunkable project? detail page?",
```
Comment (shaun-technovation) [inline app/views/team_submissions/pieces/development_platform.en.html.erb:102]: Looks like a typo here w/ an extra `?`? 🤔 😄 

> Thunkable project? detail page?

File: spec/features/mentor/submission_dev_platform_spec.rb [hunk 2]
```diff
@@ -57,7 +57,7 @@
     fill_in "What is the email address of your team's Thunkable account (optional)?",
       with: "our-team@thunkable.com"
 
-    fill_in "What is the URL to your Thunkable project?",
+    fill_in "What is the URL to your Thunkable project? detail page?",
```
Comment (shaun-technovation) [inline spec/features/mentor/submission_dev_platform_spec.rb:60]: Ah, this one too, and the ones below it. 😄

**Review by shaun-technovation — APPROVED** [review]
Besides the minor feedback, LGTM! 👍 🙌 :shipit: 🚀

File: spec/features/mentor/submission_dev_platform_spec.rb (hunk 2 above)
Comment (viviancan) [reply spec/features/mentor/submission_dev_platform_spec.rb]: nice catch! ty!

**Pull request merged**